tests/python_package_test/test_engine.py

```diff
@@ -14,6 +14,7 @@
                               load_iris, load_svmlight_file)
 from sklearn.metrics import log_loss, mean_absolute_error, mean_squared_error, roc_auc_score
 from sklearn.model_selection import train_test_split, TimeSeriesSplit, GroupKFold
+from sklearn.datasets import make_multilabel_classification
```

Comment:
There is already `from sklearn.datasets import ...` at L13.

Reply:
moved there, done!